Bottom: 03680dc4b78eb266f4a17680ca4b0491d6dce3bf
Top:    457f97bc246306ee3d3e53d49e3d3bff3680ee55
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-10 13:46:31 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
